Sources/TokamakCore/Modifiers/TextFieldStyle.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Jed Fox on 06/30/2020.
+//
+
+public protocol TextFieldStyle {
+  init()
+}
+
+public struct DefaultTextFieldStyle: TextFieldStyle {
+  public init() {}
+}
+
+public struct PlainTextFieldStyle: TextFieldStyle {
+  public init() {}
+}
+
+public struct RoundedBorderTextFieldStyle: TextFieldStyle {
+  public init() {}
+}
+
+public struct SquareBorderTextFieldStyle: TextFieldStyle {
+  public init() {}
+}
+
+extension TextField where Label == Text {
+  public func textFieldStyle(_ style: TextFieldStyle) -> Self {
+    _TextFieldProxy(self).with(style: style).subject
```

Comment:
**suggestion:** No need to use a proxy within `TokamakCore` as you have access to all these properties and types directly. The only purpose of proxies is to give access to internal properties to modules other than `TokamakCore`, like `TokamakDOM`.

```suggestion
    TextField(from: self, textFieldStyle: style)
```